evm/verify-contract.js

```diff
@@ -320,6 +330,46 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'AxelarAmplifierGateway': {
+            let args = options.args;
+
+            if (args) {
+                args = JSON.parse(args);
+                options = { ...options, ...args };
+            }
+
+            const AxelarAmplifierGateway = require('@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts/gateway/AxelarAmplifierGateway.sol/AxelarAmplifierGateway.json');
+            const amplifierGatewayFactory = await getContractFactoryFromArtifact(AxelarAmplifierGateway, wallet);
+            const amplifierGateway = amplifierGatewayFactory.attach(options.address || chain.contracts.AxelarGateway.address);
+
+            const implementation = await amplifierGateway.implementation();
+            const previousSignersRetention = (await amplifierGateway.previousSignersRetention()).toNumber();
+            const domainSeparator = await amplifierGateway.domainSeparator();
+            const minimumRotationDelay = (await amplifierGateway.minimumRotationDelay()).toNumber();
+
+            options.keyID =
+                chain.contracts.AxelarGateway.startingKeyIDs[0] || options.keyID || `evm-${chain.axelarId.toLowerCase()}-genesis`;
+            const signerSets = await getWeightedSigners(config, chain, options);
+            const operator = options.operator || chain.contracts.AxelarGateway.operator || wallet.address;
+            const owner = options.owner || chain.contracts.InterchainGovernance?.address || wallet.address;
+
+            const setupParams = defaultAbiCoder.encode([`address`, `${WEIGHTED_SIGNERS_TYPE}[]`], [operator, signerSets]);
+
+            verifyOptions.contractPath = 'contracts/gateway/AxelarAmplifierGateway.sol:AxelarAmplifierGateway';
```

Comment:
shouldn't be needed

Reply:
@blockchainguyy That indicates an issue with the gmp-sdk repo. Can you modify the test contract so it's bytecode is different from the main one?

Reply:
Seems to be a issue with Hardhat plugin. Getting this error `NomicLabsHardhatPluginError: More than one contract was found to match the deployed bytecode.`  on fantom but no issue in verifying on other chains like Avalanche, Optimism.
Instead of changing the test code as bytecode is different then the original contract, will raise issue with Hardhat team